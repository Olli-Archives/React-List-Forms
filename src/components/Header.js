import React from 'react';
import PropTypes from 'prop-types';

export default function Header({ title }){
  console.log('title in header',  title);
  return <li><h1>{title}</h1></li>;
}

Header.propTypes = {
  title: PropTypes.string.isRequired
}; 
