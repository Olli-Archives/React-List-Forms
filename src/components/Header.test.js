import Header from './Header';
import { shallow } from 'enzyme';
import React from 'react';

describe('Header component', ()=>{
  it('creates a header', ()=>{
    const title = 'test header';
    const wrapper = shallow(<Header title={title}/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
