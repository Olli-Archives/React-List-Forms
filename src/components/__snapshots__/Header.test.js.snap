// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component creates a header 1`] = `
<li>
  <h1>
    test header
  </h1>
</li>
`;
